app/aws-lsp-codewhisperer-runtimes/package.json

```diff
@@ -6,7 +6,7 @@
     "scripts": {
         "clean": "rm -rf out/ bin/ tsconfig.tsbuildinfo",
         "compile": "tsc --build",
-        "copy:wasm": "copyfiles -f --error ../../node_modules/@aws/fully-qualified-names/node/aws_fully_qualified_names_bg.wasm ../../node_modules/web-tree-sitter/tree-sitter.wasm out",
+        "copy:wasm": "copyfiles -f --error ../../node_modules/web-tree-sitter/tree-sitter.wasm out",
```

Comment:
I think this as a whole could be removed as well since it's from the fqn dependency as well, but we could do is as a follow up